src/clojuratica/jlink.clj

```diff
@@ -5,33 +5,33 @@
   ;Thus, you'll this library required but inused across the codebase.
   This is to get around that fact that we don't have the jar available to us through a standard maven repository, and can't use environment variables in our `deps.edn` specifications."
   (:require [cemerick.pomegranate :as pom]
-            [clojuratica.lib.debug :as debug]
-            [clojuratica.lib.options :as options]
-            [clojuratica.runtime.dynamic-vars :as dynamic-vars]
             [clojure.java.io :as io]
             [clojure.string :as str]))
 
 (def ^:private unable-to-find-message "Unable to find Mathematica installation. Please specify using either the MATHEMATICA_INSTALL_PATH or WOLFRAM_INSTALL_PATH environment variable.")
 
-(def ^:private default-mac-base-path "/Applications/Mathematica.app/Contents")
+(def ^:private default-mac-base-path "/Applications/Wolfram Engine.app/Contents/Resources/Wolfram Player.app/Contents")
+(def ^:private mathematica-mac-base-path "/Applications/Mathematica.app/Contents")
```

Comment:
there are two options available for mac here
* wolfram engine (which I think is free)
* mathematica 

So it was made to default to WE